README.md

```diff
@@ -13,35 +13,45 @@ GPIO Micro Service - device service for connecting GPIO devices to EdgeX
 
 ## Usage
 - This Device Service runs with other EdgeX Core Services, such as Core Metadata, Core Data, and Core Command
-- The gpio device service can contains many pre-defined devices which were defined by `configuration.toml` such as `Custom-GPIO-Device`. These devices are created by the GPIO device service in core metadata when the service first initializes
-- Device profiles are used to describe the actual GPIO hardware of a device and allow individual gpios to be given human-readable names/aliases
+- The gpio device service can contains many pre-defined devices which were defined by `res/devices/device.custom.gpio.toml` such as `GPIO-Device01`. These devices are created by the GPIO device service in core metadata when the service first initializes
+- Device profiles(`/res/profiles/device.custom.gpio.yaml`) are used to describe the actual GPIO hardware of a device and allow individual gpios to be given human-readable names/aliases
 - After the gpio device service has started, we can read or write these corresponding pre-defined devices
 
 ```yaml
```

Comment:
> suggest to link the /res/profiles/device.custom.gpio.yaml file instead of duplicate the content on the README

Already fixed in this commit(a34b231)